allow negative offset (#8487)
prometheus/prometheus: docs/querying/basics.md

    rate(http_requests_total[5m] offset 1w)

For comparisons with temporal shifts forward in time, a negative offset
can be specified:

    rate(http_requests_total[5m] offset -1w)

> This negative offset has been disabled by default since it breaks the invariant

Reviewer: ```suggestion
This negative offset is disabled by default since it breaks the invariant
```

Reviewer: I think the detail about the invariant should go to the disabled feature page.
